rib/rib_test.go

```diff
@@ -3358,10 +3358,8 @@ func TestFlush(t *testing.T) {
 			r := NewRIBHolder("DEFAULT")
 
 			_, _, err := r.AddNextHop(&aftpb.Afts_NextHopKey{
-				Index: 1,
-				NextHop: &aftpb.Afts_NextHop{
-					ProgrammedIndex: &wpb.UintValue{Value: 42},
```

Comment:
Here and below, is there another field that we can populate within the `NextHop` just to check that it is being populated correctly and returned in the RIB?

Reply:
Done.

Reply:
Done